LanguageFeatures/Extension-types/nullability_A01_t02.dart

```diff
@@ -2,10 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion Let V be an extension type of the form Name<T1, .. Ts>, and let R
-/// be the corresponding instantiated representation type. If R is non-nullable
-/// then V is a proper subtype of Object, and V is non-nullable. Otherwise, V is
-/// a proper subtype of Object?, and V is potentially nullable.
+/// @assertion An extension type `V` is a proper subtype of `Object?`. It is
+/// potentially non-nullable, unless it implements `Object` or a subtype thereof
```

Comment:
Leaf mentioned that this wording isn't precise. It should have said "It is potentially non-nullable.If it implements `Object` or a subtype thereof then it is also non-nullable."

This will be fixed soon.

Reply:
I'm not sure which approach is best when it comes to adjusting the '@assertion' (because the feature specification hasn't been fixed yet). WDYT?

Reply:
Let's update it later when the spec will be actually updated.

Reply:
OK.